### PR TITLE
Remove auto_tag from .drone.yml

### PR DESCRIPTION
**Pull request by zeripath**
auto_tag causes drone to tag tags with latest.

Signed-off-by: Andrew Thornton <art27@cantab.net>

---
hotfix for #20058

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -902,8 +902,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-amd64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
```
Comment (6543) [inline .drone.yml:905]: I thought we use manifest to create latest tag:
https://github.com/go-gitea/gitea/blob/main/docker/manifest.tmpl#L6-L8

Comment (zeripath) [reply .drone.yml:905]: the auto_tag is what is putting those there.

**Comment by 6543** [comment]
Instead of have to enable/disable stuff per commit - why not check with a regex, that the tag meet the `latest`-tag requirements within manifest tmpl ?

**Comment by 6543** [comment]
if tag match `^v\d+.\d+.\d$` - then it's also latest ...

File: .drone.yml (hunk 1 above)
Comment (6543) [inline .drone.yml:905]:
```suggestion
      # Enable auto_tag: true on the latest release branch only
```

File: .drone.yml [hunk 2]
```diff
@@ -920,8 +921,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     settings:
       dockerfile: Dockerfile.rootless
-      auto_tag: true
-      auto_tag_suffix: linux-amd64-rootless
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
```
Comment (6543) [inline .drone.yml:924]:
```suggestion
      # Enable auto_tag: true on the latest release branch only
```

File: .drone.yml [hunk 3]
```diff
@@ -1126,8 +1128,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-arm64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
```
Comment (6543) [inline .drone.yml:1131]:
```suggestion
      # Enable auto_tag: true on the latest release branch only
```

File: .drone.yml [hunk 4]
```diff
@@ -1144,8 +1147,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     settings:
       dockerfile: Dockerfile.rootless
-      auto_tag: true
-      auto_tag_suffix: linux-arm64-rootless
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
```
Comment (6543) [inline .drone.yml:1150]:
```suggestion
      # Enable auto_tag: true on the latest release branch only
```

File: .drone.yml [hunk 5]
```diff
@@ -902,8 +902,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-amd64
```
Comment (6543) [inline .drone.yml:906]: why remove tag suffix?

File: .drone.yml [hunk 6]
```diff
@@ -1299,7 +1303,7 @@ steps:
     image: plugins/manifest
     pull: always
     settings:
-      auto_tag: true
+      auto_tag: false
```
Comment (6543) [inline .drone.yml:1306]: add note here too ...

File: .drone.yml [hunk 7]
```diff
@@ -1310,7 +1314,7 @@ steps:
   - name: manifest
     image: plugins/manifest
     settings:
-      auto_tag: true
```
Comment (6543) [inline .drone.yml:1313]: add note here too ...

**Review by 6543 — APPROVED** [review]

**Review by 42wim — APPROVED** [review]

File: .drone.yml [hunk 8]
```diff
@@ -902,8 +902,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-amd64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
+      auto_tag: false
+      tags: ${DRONE_TAG}-linux-amd64
```
Comment (6543) [inline .drone.yml:907]:
```suggestion
      tags: ${DRONE_TAG##v}-linux-amd64
```
[I think ](https://gitea.com/gitea/drone-docker/src/branch/master/tags.go#L61-L71) get's trimed on auto_tag - we have to imitate it

Comment (6543) [inline .drone.yml:907]:
```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      # auto_tag_suffix: linux-amd64
      tags: ${DRONE_TAG##v}-linux-amd64
```

File: .drone.yml [hunk 9]
```diff
@@ -920,8 +921,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     settings:
       dockerfile: Dockerfile.rootless
-      auto_tag: true
-      auto_tag_suffix: linux-amd64-rootless
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
+      auto_tag: false
+      tags: ${DRONE_TAG}-linux-amd64-rootless
```
Comment (6543) [inline .drone.yml:926]:
```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      # auto_tag_suffix: linux-amd64-rootless
      tags: ${DRONE_TAG##v}-linux-amd64-rootless
```

File: .drone.yml [hunk 10]
```diff
@@ -1126,8 +1128,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     pull: always
     settings:
-      auto_tag: true
-      auto_tag_suffix: linux-arm64
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
+      auto_tag: false
+      tags: ${DRONE_TAG}-linux-arm64
```
Comment (6543) [inline .drone.yml:1133]:
```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      # auto_tag_suffix: linux-arm64
      tags: ${DRONE_TAG##v}-linux-arm64
```

File: .drone.yml [hunk 11]
```diff
@@ -1144,8 +1147,9 @@ steps:
     image: techknowlogick/drone-docker:latest
     settings:
       dockerfile: Dockerfile.rootless
-      auto_tag: true
-      auto_tag_suffix: linux-arm64-rootless
+      # Either set auto_tag: true or add latest to tags once 1.17.0 is released
+      auto_tag: false
+      tags: ${DRONE_TAG}-linux-arm64-rootless
```
Comment (6543) [inline .drone.yml:1152]:
```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      # auto_tag_suffix: linux-arm64-rootless
      tags: ${DRONE_TAG##v}-linux-arm64-rootless
```

File: .drone.yml (hunk 6 above)
Comment (6543) [inline .drone.yml:1306]:
```suggestion
      # Enable on the latest release branch only and tags
      # auto_tag: true
      tags: ${DRONE_TAG}
```
this time we dont trim as we have https://github.com/go-gitea/gitea/blob/main/docker/manifest.tmpl#L11

**Pull request closed without merging**